### ISSUE TITLE
force_ffms2 non working

### ISSUE THREAD

**Issue by kliffgomel** [R3S3t9999/DoVi_Scripts#31]
force_ffms2=true in SETTINGS.bat 
This setting does not work properly, ffms is not forced with dovi_baker.
The script tries to run DGIndex.
I am using the latest beta version of the script.

**Comment by R3S3t9999** [comment]
Hi,

This setting is for the screenshot workflow (7-1) only and it's working fine on my end so I'm guessing you're talking about the encoder workflow (8-2-1) ???  I didn't add ffms2 support there because, on my end, ffms2 decoding is soooo slow that it's practically unusable. 

`choose if you want to force ffms2 in 7-1 dovi baker for those who dont have an Nvidia GPU. default = NO
set force_ffms2=NO`

Please try 7-1 and check the performance. Let me know if it's usable on your PC and I'll add support in 8-2-1 too.

**Comment by kliffgomel** [comment]
Of course, I am talking about video encoding. It turns out that video encoding in some cases will not be available if there is no nvidia video card, in my opinion, this significantly reduces the functionality of the script.

**Comment by R3S3t9999** [comment]
right but as I said, the dovi_baker with ffms2 is so slow that it's practically unusable so that's I added support only to the screenshot workflow (7-1) and didn't waste time adding support in the encoder workflow, but please try 7-1 and let me know the performance you're getting.

**Comment by kliffgomel** [comment]
It's hard to estimate the performance from screenshots, yes, it's not high, no more than 1 frame per second. In any case, Dovi_Baker encoding is a mode for enthusiasts, performance does not play a major role here.

**Comment by R3S3t9999** [comment]
How much time to export 50 screenshots? For reference, with Nvidia dgindex it takes less than 5 minutes(about 20-23fps) while with ffms2 it takes hours on my end so I don't see how this could work with a movie that has over 150 000 frames but I'll add support in 8-2-1 anyway in the next beta tonight.

**Comment by kliffgomel** [comment]
thanks!

**Comment by R3S3t9999** [comment]
The [latest Beta](https://drive.google.com/file/d/128gq8aDUTKA_aT7SQsM9dkjA1EP1sosR/view?usp=drive_link) is available now!
